apps/server/src/AppService.ts

```diff
@@ -33,4 +39,54 @@ export class AppService {
     const eventSignature = this.contract.filters.BRIDGE_TO_DEFI_CHAIN();
     return this.contract.queryFilter(eventSignature, blockNumber, currentBlockNumber - 65);
   }
+
+  async signClaim({ receiverAddress, tokenAddress, amount }: SignClaim): Promise<{ signature: string; nonce: number }> {
+    try {
+      // Connect signer ETH wallet (admin/operational wallet)
+      const wallet = new ethers.Wallet(this.configService.getOrThrow('ethereum.testnet.ethWalletPrivKey'));
+      const signer = wallet.connect(this.ethersRpcProvider);
```

Comment:
```suggestion
      const wallet = new ethers.Wallet(this.configService.getOrThrow('ethereum.testnet.ethWalletPrivKey'), this.ethersRpcProvider);
```

We can skip the redundant `connect` call by passing it in directly to the constructor.